Dockerfile

```diff
@@ -9,9 +9,12 @@ ENV NEXT_TELEMETRY_DISABLED=1
 
 WORKDIR /app
 COPY . ./
-RUN yarn install --immutable && \
+ENV NODE_ENV=production
```

Comment:
The `ENV` statement can be moved one instruction up in the Dockerfile. This increases the possibilities of the build cache to work as even [`ENV` statements create new layers](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#:~:text=Each%20ENV%20line%20creates%20a,following%2C%20and%20then%20building%20it.).